scripts/rasterizer.js

```diff
@@ -78,10 +81,25 @@ service = server.listen(port, function(request, response) {
     if (request.headers.clipRect) {
       page.clipRect = JSON.parse(request.headers.clipRect);
     }
-    for (name in pageSettings) {
-      if (value = request.headers[pageSettings[name]]) {
+    pageSettings.forEach(function(setting) {
+      var value = request.headers[setting];
+      if (value) {
         value = (value == 'false') ? false : ((value == 'true') ? true : value);
-        page.settings[pageSettings[name]] = value;
+        page.settings[setting] = value;
+      }
+    });
+    if (path.substr(-4) === '.pdf') {
+      //Example values for pdf param: "5in*7.5in", "10cm*20cm", "A4", "Letter"
+      var pdf = request.headers.pdf || defaultPDF;
+      var wxh = pdf.split('*');
+      page.paperSize = wxh.length === 2 ? {
```

Comment:
Don't think I understand this question.   The variable 'pdf' is passed to screenshot via parameter.  This variable determines the paper-size of the PDF.